site/versioning-policy.html

```diff
@@ -187,9 +187,9 @@ <h3>Alpha components</h3>
 
 <h3>API compatibility</h3>
 
-<p>An API is any public class or interface exposed in Spark that is not marked as &#8220;developer API&#8221; or 
-&#8220;experimental&#8221;. Release A is API compatible with release B if code compiled against release A 
-<em>compiles cleanly</em> against B. Currently, does not guarantee that a compiled application that is 
+<p>In general, An API is any public class or interface documented in Spark.
```

Comment:
Hmm, what does it exactly mean for "documented"? Just with code comment? Or required to have public API doc in the official site like https://spark.apache.org/docs/latest/api/scala/org/apache/spark/index.html?

Reply:
It is just said "documented" that might be a little confused.

Reply:
yeah, I meant that documented in everywhere in Spark.